src/Stnvh/Partial/Zip.php

```diff
@@ -64,9 +64,6 @@ private function receiveData($ch, $data) {
 	 * @return void
 	 */
 	public function __construct($url, $file = false) {
-		set_time_limit(15);
-		ob_start();
-
```

Comment:
Will merge PHP 5.3 fixes, and manually refactor error handling soon, including checks for ```Accept-Ranges: bytes```